lesson10/index.js

```diff
@@ -0,0 +1,59 @@
+const ul = document.getElementById('js-lists');
+
+const listData = [
+    {to: "bookmark.html", img: "1.png", alt:"画像1", text: "ブックマーク"},
+    {to: "message.html", img: "2.png", alt:"画像2", text: "メッセージ"}
+]
+
+function displayLoadingImage () {
+    const lodingImage = document.createElement('img');
```

Comment:
nits
I think it's “loading”.
I often misspell it too.
I have added an extension to VScode.
Plugins I use [this](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker)